core/org.wso2.carbon.server/src/main/java/org/wso2/carbon/server/util/FileUtils.java

```diff
@@ -259,15 +259,15 @@ public static boolean deleteDir(File dir) {
     }
 
 
-	public static File getFile(File parentFile,final String fileName) {
-		File[] files = parentFile.listFiles();
-		for ( File file : files ) {
-			if (file.getName().equals(fileName)) {
-				return file;
-			}
-		}
-		return null;
-	}
+    public static File getFile(File parentFile, final String fileName) {
```

Comment:
Doc comment for the public method

Reply:
Fixed - https://github.com/wso2/carbon-kernel/pull/2477